assignments/models.py

```diff
@@ -59,6 +67,70 @@ class Meta:
         verbose_name = _('open text task')
         verbose_name_plural = _('open text tasks')
 
+
+class BudgetTextTask(models.Model):
+    """
+    Budget text task. Budget si related to section and consists of one or more Budgeting targets
+    """
+    UNIT_HA = 0
+    UNIT_PCS = 1
+    UNIT_CHOICES = (
+        (UNIT_HA, _('ha')),
+        (UNIT_PCS, _('pcs'))
+    )
+
+    section = models.ForeignKey(Section, related_name='budget_text_tasks')
+    name = models.CharField(_('name'), max_length=256)
+    unit = models.IntegerField(choices=UNIT_CHOICES, default=UNIT_HA)
+
+    class Meta:
+        verbose_name = _('budget text task')
+        verbose_name_plural = _('budget text tasks')
+
     def __str__(self):
-        return 'Question: {}'.format(self.question)
+        return self.name
+
+
+class BudgetTextTarget(models.Model):
+    """
+    Budget text task targets.
+    """
 
+    task = models.ForeignKey(BudgetTextTask, related_name='budget_targets')
```

Comment:
It might actually be handy to reuse the `BugdetTargets` between tasks.. So for example `Suburb A` assignment could have budgeting task with `Park type A` and `Park type B` `BudgetTargets` and then another assignment `Suburb B` could use those same `BudgetTargets`.